CHANGELOG.rst

```diff
@@ -117,6 +117,8 @@ Fixed
   Contributed by Tatsuma Matsuki (@mtatsuma)
 
 * Fix dependency conflicts by updating ``requests`` (2.23.0) and ``gitpython`` (2.1.15). #4869
+* Fix orquesta syntax error for with items task where action is misindented or missing.
+  PR @ https://github.com/StackStorm/orquesta/pull/195 (bug fix)
```

Comment:
I don't remember we included links to external PRs, but if we want, the following would look more compact:
```suggestion
  StackStorm/orquesta#195 (bug fix)
```
and transformed to StackStorm/orquesta#195

Reply:
Fixed